instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/sessionreplay/BaseSessionReplayTest.kt

```diff
@@ -13,6 +13,8 @@ import androidx.test.platform.app.InstrumentationRegistry
 import com.datadog.android.Datadog
 import com.datadog.android.rum.GlobalRumMonitor
 import com.datadog.android.sdk.integration.RuntimeConfig
+import com.datadog.android.sdk.integration.sessionreplay.BaseSessionReplayTest.MatchingStrategy.CONTAINS
+import com.datadog.android.sdk.integration.sessionreplay.BaseSessionReplayTest.MatchingStrategy.EXACT
```

Comment:
It is better to keep only `import com.datadog.android.sdk.integration.sessionreplay.BaseSessionReplayTest.MatchingStrategy` and then reference members through enum name in order to be aligned with the code style used in the project.

Reply:
IDE will always add these two lines when optimizing imports, no matter what I do. I think it's because the enum class is in the same file as this class

Reply:
That is probably because its rules are different from `Ktlint`, you need to adjust it:

![image](https://github.com/user-attachments/assets/7e7922b4-39f1-41a0-ac71-bfda04db4480)

I personally don't use `Optimize imports` functionality and rely completely on `ktlint` for formatting.

Reply:
the change on imports config still doesn't work, I disabled the "save actions" for this comment to remove these two lines.